en/releases/release-1.1.7.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TiDB Operator 1.1.7 Release Notes
+---
+
+# TiDB Operator 1.1.7 Release Notes
+
+Release date: November 13, 2020
+
+TiDB Operator version: 1.1.7
+
+## New Features
+
+- Support `spec.toolImage` for Backup & Restore with BR to define the image used to provide the BR binary executables, use `pingcap/br:${tikv_version}` by default ([#3471](https://github.com/pingcap/tidb-operator/pull/3471), [@namco1992](https://github.com/namco1992))
```

Comment:
```suggestion
- Support `spec.toolImage` for the `Backup` and `Restore` CR with BR to define the image used to provide the BR binary executables. Defaults to `pingcap/br:${tikv_version}` ([#3471](https://github.com/pingcap/tidb-operator/pull/3471), [@namco1992](https://github.com/namco1992))
```
`Backup & Restore` is also the full name of `BR`... let's make sure readers don't get confused.